src/consensus/consensus.h

```diff
@@ -28,4 +30,8 @@ enum {
     LOCKTIME_MEDIAN_TIME_PAST = (1 << 1),
 };
 
+static const int64_t SECONDS_PER_YEAR = 31558060;
```

Comment:
For some reason, that gets `ctime(47*(31556952))` => 'Sat Dec 31 09:32:24 2016'

I agree your math looks more correct, but for some reason there are more seconds to get the desired result.